packages/api/emails/verifyHcEmail/html.pug

```diff
@@ -1,5 +1,6 @@
 div.main
-  img(src=`${appURL}/email/header.jpg` alt='header' style='width:100%; height:auto;')
+  //- img(src='cid:ksf-email-header' alt='header' style='width:100%; height:auto;')
```

Comment:
nit: You can remove this commented out line

Reply:
Removed in latest commit